ros_controllers/package.xml

```diff
@@ -23,6 +23,7 @@
   <exec_depend>effort_controllers</exec_depend>
   <exec_depend>force_torque_sensor_controller</exec_depend>
   <exec_depend>forward_command_controller</exec_depend>
+  <exec_depend>four_wheel_steering_controller</exec_depend>
```

Comment:
This decision is up to @bmagyar

Reply:
Yes this is intended. The controller is in the repo, let's go all the way.